### PR TITLE
Feature/cu mfce60 gamepad control

### PR DESCRIPTION
**Pull request by devildrake**
Gamepad control established
Centered the way all inputs are used inside Player.h defining InputActions:
	MOVEMENT,  ORIENTATION,  SWITCH, SHOOT, ABILITY_1, ABILITY_2, ABILITY_3

Movement and Orientation actions are handled through a method returning float2
All other InputActions are handled through a method returning Bool

Gamepad control will be used if the bool inside PlayerController.h named useGamepad is true and a Gamepad is connected
useGamePad bool inside PlayerController can be set through a public method SetUseGamepad()
This method should be called from some UI button
Orientation is slightly more complicated, established a way for gamepad and mouse behaviors to coexist

### CODE REVIEW

**Comment by jcarlos0305** [comment]
Task linked: [CU-mfce60 Player control with gamepad](https://app.clickup.com/t/mfce60)

**Review by Jordi-Gil — DISMISSED** [review]

File: Penteract/Assets/Scripts/UseControllerSetter.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include "UseControllerSetter.h"
+#include "GameObject.h"
+#include "GameplaySystems.h"
+#include "PlayerController.h"
+
+EXPOSE_MEMBERS(UseControllerSetter) {
+	// Add members here to expose them to the engine. Example:
```
Comment (isaacfdez) [inline Penteract/Assets/Scripts/UseControllerSetter.cpp:7]: Delete comments

File: Penteract/Assets/Scripts/UseGamepadSetter.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#include "UseGamepadSetter.h"
+#include "PlayerController.h"
+
+EXPOSE_MEMBERS(UseGamepadSetter) {
+	// Add members here to expose them to the engine. Example:
```
Comment (isaacfdez) [inline Penteract/Assets/Scripts/UseGamepadSetter.cpp:5]: Delete comments

File: Penteract/Assets/Scripts/UseControllerSetter.cpp (hunk 1 above)
Comment (devildrake) [reply Penteract/Assets/Scripts/UseControllerSetter.cpp:7]: Done

File: Penteract/Assets/Scripts/UseGamepadSetter.cpp (hunk 2 above)
Comment (devildrake) [reply Penteract/Assets/Scripts/UseGamepadSetter.cpp]: Done

**Review by isaacfdez — APPROVED** [review]

**Review by jcarlos0305 — DISMISSED** [review]

**Review by isaacfdez — APPROVED** [review]

**Review by isaacfdez — APPROVED** [review]

**Pull request merged**